docs/_extra/api-reference/schemas/annotation.yaml

```diff
@@ -0,0 +1,87 @@
+Annotation:
+  type: object
+  required:
+    - id
+    - created
+    - updated
+    - user
+    - uri
+    - text
+    - tags
+    - group
+    - permissions
+    - target
+    - document
+    - links
+    - hidden
+    - flagged
+  properties:
+    id:
+      type: string
+    created:
+      type: string
+      format: date-time
+    updated:
+      type: string
+      format: date-time
+    user:
+      type: string
+      pattern: "acct:^[A-Z0-9._]{3,30}@.*$"
```

Comment:
Could this reuse the definition of the `UserID` field that is defined elsewhere? Same question about referencing group IDs.